lib/driver.rb

```diff
@@ -3,8 +3,16 @@
 require './TestScriptRunnable' # TODO: Remove
 
 default_engine = TestScriptEngine.new 'http://hapi.fhir.org/baseR4'
+<<<<<<< Updated upstream
 default_engine.load_scripts
 # default_engine.load_scripts(nil, 'update_test_script_obs.json')
+=======
+# 'http://test.fhir.org/r4/' 
+# Load and execute all TestScript in /TestScripts
+# default_engine.load_scripts
+# Specify a TestScript to execute
+default_engine.load_scripts(nil, 'general_test_script.json')
+>>>>>>> Stashed changes
```

Comment:
Since driver.rb is an execution code for the engine, unless we have fundamental change of it (change of arguments, etc.), it may be good to exclude from commit. I will exclude it from future commit.